ia/src/load.py

```
@@ -0,0 +1,269 @@
+        handler(client, response)
+
+
+def execute_command(client, command, args) -> None:
+    global allowed_commands
+
+    if command not in allowed_commands:
+        raise ValueError(f"Command '{command}' non autorisée")
+    if command == utils.BROADCAST or command == utils.SET or command == utils.TAKE:
+        send_message(client, f"{command} {args}")
+    elif command == utils.FORWARD:
+        if client.last_look[utils.PLAYER_CELL].count("player") > utils.CANT_MOVE and client.help_status:
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)